lib/danger/danger_core/dangerfile.rb

```diff
@@ -44,13 +44,15 @@ def core_plugin_classes
     # http://ruby-doc.org/core-2.2.3/Kernel.html#method-i-fail
     # However, as we're using using them in the DSL, they won't
     # get method_missing called.
+    
+    # maybe?
 
-    def warn(message)
-      method_missing(:warn, message)
+    def warn(message, sticky: true)
```

Comment:
Maybe these should take `*args, &blk` and just forward those to `method_missing`?